### PR TITLE
feat(integration): Singular- allow empty string by default for instal…

### PR DESCRIPTION
**Pull request by yashasvibajpai**
…l_source

## Description of the change

> install_source, a property for android session events is allowed to have empty string as value

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: __tests__/data/singular.json [hunk 1]
```diff
@@ -478,117 +478,6 @@
       }
     }
   },
-  {
-    "description": "[Error]: (Android) Required field referring application missing for Session event",
```
Comment (utsabc) [inline __tests__/data/singular.json:482]: Why do we need to remove this test case?

Comment (yashasvibajpai) [reply __tests__/data/singular.json:482]: It was error case since this install_source was mandatory required earlier, so for missing it

Comment (utsabc) [reply __tests__/data/singular.json:482]: But we can keep this case and rather than error we can have payload with `""` in it

Comment (yashasvibajpai) [reply __tests__/data/singular.json:482]: Okay fixing this case.

**Review by utsabc — APPROVED** [review]

**Review by akashrpo — APPROVED** [review]

**Pull request merged**